pkg-manifest/exportable-manifest/src/index.ts

```diff
@@ -100,3 +108,37 @@ async function makePublishDependency (depName: string, depSpec: string, dir: str
   }
   return depSpec
 }
+
+async function makePublishPeerDependency (depName: string, depSpec: string, dir: string, modulesDir?: string) {
+  if (!depSpec.includes('workspace:')) {
+    return depSpec
+  }
+
+  // Dependencies with bare "*", "^", "~",">=",">","<=",< versions
+  const workspaceSemverRegex = /workspace:([\^~*]|>=|>|<=|<)/
+  const versionAliasSpecParts = workspaceSemverRegex.exec(depSpec)
+
+  if (versionAliasSpecParts != null) {
+    modulesDir = modulesDir ?? path.join(dir, 'node_modules')
+    const manifest = await resolveManifest(depName, modulesDir)
+
+    const [,semverRangGroup] = versionAliasSpecParts
+
+    const semverRangeToken = semverRangGroup !== '*' ? semverRangGroup : ''
+
+    return depSpec.replace(workspaceSemverRegex, `${semverRangeToken}${manifest.version}`)
+  }
+
+  if (depSpec.includes('workspace:./') || depSpec.includes('workspace:../')) {
```

Comment:
- [ ] Need to write a test case for this. Does seem to be one already for reference (that is not skipped)

Reply:
Took this out. Realized this is essentially just an alias as well when I cam back and looked at it